src/app/components/tabs/index.html

```diff
@@ -1,15 +1,292 @@
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot screenshot-danger">
+					<img class="bordered" src="~/assets/img/guidelines/tab/dont-overflow.png" />
+					<div class="caption sky-text-danger">Don't use horizontal tabs when they are likely to overflow.</div>
+				</div>
+			</div>
+		</div>
+		
+	</sky-demo-page-content>
+	
+	<sky-demo-page-content sectionHeading="Anatomy">
+		<div class="two-col-right-image sky-margin-stacked-separate">
+			<div class="two-col-text">
+				<div class="anatomy-list">
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">1</div><div class="anatomy-description">Selected tab</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">2</div><div class="anatomy-description">Unselected tab</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">3</div><div class="anatomy-description">Tab border</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">4</div><div class="anatomy-description">Tab count (optional)</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">5</div><div class="anatomy-description">Close tab button (optional)</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">6</div><div class="anatomy-description">Open tab button (optional)</div></div>
+				</div>
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot">
+					<img class="bordered" srcset="~/assets/img/guidelines/tab/anatomy.png 1x, ~/assets/img/guidelines/tab/anatomy_2x.png 2x" src="~/assets/img/guidelines/tab/anatomy.png" />
+				</div>
+			</div>
+		</div>
+	</sky-demo-page-content>
+	
+	<sky-demo-page-content sectionHeading="Options">
+		<h3>Count</h3>
+			<div class="two-col-right-image sky-margin-stacked-separate">
+				<div class="two-col-text">
+					<p>When you use tabs to hold lists, you can include the list's record count in the tab so users know if there are records to look at without navigating to the tab.</p>
+				</div>
+				<div class="two-col-image">
+					<div class="screenshot">
+						<img class="bordered" src="~/assets/img/guidelines/tab/option-count.png" />
+					</div>
+				</div>
+			</div>
+		<h3>Open and close</h3>
+			<div class="two-col-right-image sky-margin-stacked-separate">
+				<div class="two-col-text">
+					<p>If users need to work with a subset of large number of available tabs, let them open and close tabs so they are only working with relevant tabs. If users have more than one option of what to display in a new tab, use action buttons within the tab view to let them choose the option they want.</p>
+				</div>
+				<div class="two-col-image">
+					<div class="screenshot">
+						<img class="bordered" src="~/assets/img/guidelines/tab/option-open-close.png" />
```

Comment:
Should this image have a caption? It seems like maybe it should have something along the lines of "Do use action buttons to enable users to select what to display in new tabs." Or something like that.